apis/sgv2-restapi/CONFIGURATION.md

```diff
@@ -8,9 +8,10 @@
 
 *Configuration for distinct endpoints*
 
-| Property                     | Type      | Default | Description                                         |
-|------------------------------|-----------|---------|-----------------------------------------------------|
-| `stargate.rest.cql.disabled` | `boolean` | `true`  | Whether /v2/cql endpoint should be disabled or not. |
+| Property                         | Type      | Default | Description                                                                                                                                                                                                                                                                                                                                                                                                        |
+|----------------------------------|-----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `stargate.rest.cql.disabled`     | `boolean` | `true`  | Whether /v2/cql endpoint should be disabled or not.                                                                                                                                                                                                                                                                                                                                                                |
+| `stargate.rest.compact-map-data` | `boolean` | `true`  | When enabled, the map data will be of the below format for example:<br/>```characteristics: { "accuracy" : "medium" }``` <br/>when disabled:<br/>```characteristics: [ { "key" : "accuracy", "value" : "medium" } ]```<br/>_This flag can be overridden in the runtime using `compactMapData` flag in the data read and write APIs. Please check the API specification in the swagger-ui for API specific options_ |
```

Comment:
Does this default to `true`? I thought we were thinking of defaulting to `false`.

Reply:
This will be `true` by default in the OSS version and this will be turned off using the flag on our deployment.

Reply:
Oh ok.